### DISCUSSION TITLE
Build Errors Related to @material-ui/core

### DISCUSSION THREAD

**Discussion by JennaSys** [rtp-book/code#4] [Errata]
If you are experiencing problems getting the project to build with errors related to `@material-ui/core`, try changing the dependency in the [package.json](https://github.com/rtp-book/project/blob/main/client/package.json) file to remove the caret (^) from the version number for that library so that npm doesn't install a later version:

`"@material-ui/core": "4.11.0"`

There was apparently a breaking change in version 4.12.0 in the `styles` module that would otherwise require refactoring of the project source code.

**Comment by mspalin** [comment]
I am using @material-ui/core 4.12.4 and got this error:
Cannot resolve dependency '@material-ui/core/styles/createMuiTheme'

Apparently createMuiTheme was deprecated in v4.12.0, so I changed all references of createMuiTheme to createTheme.

**Reply by JennaSys** [reply]
Yes that works. I mentioned that in the[ project repo](https://github.com/rtp-book/project/discussions/4#discussioncomment-2200190) too.
